.replit

```diff
@@ -1,10 +1,6 @@
 run = "make && echo \"UPM is built, can be found here: ./cmd/upm/upm\""
 
-[nix]
-channel = "stable-21_11" # Only channel with go 1.17
-
-[languages.go]
-pattern = "**/*.go"
+modules = ["go-1.20:v2-20230911-b5aa5df"]
 
-[languages.go.languageServer]
-start = "gopls"
\ No newline at end of file
+[nix]
+channel = "stable-23_05" # Only channel with go 1.17
```

Comment:
it's actually 1.20 oops

Reply:
the comment is irrelevant, forgot to remove :facepalm:

Reply:
https://github.com/replit/upm/pull/138